index.js

```diff
@@ -186,12 +186,9 @@ var publicAgent = {
           }
         }
         if (err) {
-          // Fatal error. Disable the agent.
           logger.error('Unable to acquire the project number from metadata ' +
             'service. Please provide a valid project number as an env. ' +
-            'variable, or through config.projectId passed to start(). ' +
-            'Disabling trace agent. ' + err);
-          publicAgent.stop();
+            'variable, or through config.projectId passed to start(). ' + err);
```

Comment:
I think that without calling stop here, the agent will continue attempting to publish to the service even though no publish will succeed. We may still need a way to turn off the trace writer even if we stop allowing developers from stopping an already started agent.